cernopendata/modules/theme/assets/semantic-ui/scss/search.scss

```diff
@@ -15,6 +15,10 @@
   }
 }
 
+.ui.fluid.container {
```

Comment:
maybe give a specific `id` as well so it doesn't affect all fluid containers unless you intend to!

Reply:
Thanks for the hint. Is this better?